CHANGELOG.rst

```diff
@@ -54,6 +54,9 @@ Bug Fixes
 - `#3695 <https://github.com/pytest-dev/pytest/issues/3695>`_: Fix ``ApproxNumpy`` initialisation argument mixup, ``abs`` and ``rel`` tolerances were flipped causing strange comparsion results.
   Add tests to check ``abs`` and ``rel`` tolerances for ``np.array`` and test for expecting ``nan`` with ``np.array()``
 
+- `#3712  <https://github.com/pytest-dev/pytest/issues/3712>`_: Correctly represent the dimensions of an numpy array when calling ``repr()`` on ``approx()``.
```

Comment:
Sorry, I should have been more clear: you should add `changelog` files to the `changelog/` folder. See the README on https://github.com/pytest-dev/pytest/tree/master/changelog for more instructions. 👍

Reply:
Nevermind, I did some a small tweak in your branch and fixed the changelog notes myself. 👍